fundamental-concepts/data-structures-stdlib/index.json

```
@@ -0,0 +1,44 @@
+{
+  "title": "Data types and the DAML standard library",
+  "description": "Learn about records, sums, sets, and maps and other essential tools of the DAML standard library.",
+  "difficulty": "Beginner",
+  "time": "30 minutes",
+  "details": {
+    "assets": {
+        "host01": [
+        ]
+    },
+    "steps": [
+      {"text": "1_records.md"},
+      {"text": "2_sums.md"},
+      {"text": "3_type_synonyms.md"},
+      {"text": "4_imports.md"},
+      {"text": "5_stdlib.md"},
+      {"text": "6_lists.md"},
+      {"text": "7_sets.md"},
+      {"text": "8_maps.md"},
+      {"text": "9_optional.md"}
+    ],
+    "intro": {
+      "text": "0_intro.md",
+      "courseData": "init_background.sh"
+    },
+    "finish": {
+      "text": "99_finish.md"
+    }
+  },
+  "environment": {
+    "showide": false,
+    "uieditorpath": "/root/create-daml-app",
+    "showdashboard": false,
+    "dashboards": [{"name": "UI", "port": 3000}],
+    "terminals": [
+        {"name": "Terminal 2", "target": "host01"}
+    ],
+    "idePort": 23000,
+    "uilayout": "terminal-iframe"
```

```suggestion
    "uilayout": "terminal"
```
Like this we would have just one terminal

---

I removed one terminal in the terminal stanza, seems to have the same effect.